src/explore-education-statistics-admin/src/pages/release/content/components/ReleaseContent.tsx

```diff
@@ -265,7 +265,13 @@ const ReleaseContent = () => {
                   {release.coverageTitle} <strong>{release.yearTitle}</strong>
                 </p>
                 <Details summary={`See other releases (${releaseCount})`}>
-                  <ul className="govuk-list">
+                  <ul
+                    className="govuk-list"
+                    style={{
```

Comment:
We'd generally add a class instead of inline styles like this. To avoid duplicating the class for both places you could see if it'd be simple to create a shared component for the other releases list.

Reply:
Sounds good, I've named the component `ScrollableList` for now